src/overlays/actors/ovl_En_Weather_Tag/z_en_weather_tag.c

```diff
@@ -26,45 +42,296 @@ const ActorInit En_Weather_Tag_InitVars = {
     (ActorFunc)EnWeatherTag_Update,
     NULL,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Weather_Tag/func_80B31020.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Weather_Tag/EnWeatherTag_Destroy.s")
+void func_80B31020(EnWeatherTag* this, EnWeatherTagActionFunc actionfunc) {
```

Comment:
It's fine to not document anything but this type of function is always named _SetupAction, so it's kinda free documentation if you want
```suggestion
void EnWeatherTag_SetupAction(EnWeatherTag* this, EnWeatherTagActionFunc actionFunc) {
```